cpp/Platform.Setters.Tests/SetterTests.cpp

```diff
@@ -1,43 +1,42 @@
-﻿namespace Platform::Setters::Tests
-{
-    TEST_CLASS(SetterTests)
-    {
-        public: TEST_METHOD(ParameterlessConstructedSetterTest)
-        {
-            Setter<std::int32_t> setter = Setter<std::int32_t>();
-            Assert::AreEqual(0, setter.Result);
-        }
-
-        public: TEST_METHOD(ConstructedWithDefaultValueSetterTest)
-        {
-            Setter<std::int32_t> setter = Setter<std::int32_t>(9);
-            Assert::AreEqual(9, setter.Result);
-        }
-
-        public: TEST_METHOD(MethodsWithBooleanReturnTypeTest)
-        {
-            Setter<std::int32_t> setter = Setter<std::int32_t>();
-            Assert::IsTrue(setter.SetAndReturnTrue(1));
-            Assert::AreEqual(1, setter.Result);
-            Assert::IsFalse(setter.SetAndReturnFalse(2));
-            Assert::AreEqual(2, setter.Result);
-            Assert::IsTrue(setter.SetFirstAndReturnTrue(std::int32_t[] { 3 }));
-            Assert::AreEqual(3, setter.Result);
-            Assert::IsFalse(setter.SetFirstAndReturnFalse(std::int32_t[] { 4 }));
-            Assert::AreEqual(4, setter.Result);
-        }
-
-        public: TEST_METHOD(MethodsWithIntegerReturnTypeTest)
-        {
-            Setter<std::int32_t, std::int32_t> setter = Setter<std::int32_t, std::int32_t>(1, 0);
-            Assert::AreEqual(1, setter.SetAndReturnTrue(1));
-            Assert::AreEqual(1, setter.Result);
-            Assert::AreEqual(0, setter.SetAndReturnFalse(2));
-            Assert::AreEqual(2, setter.Result);
-            Assert::AreEqual(1, setter.SetFirstAndReturnTrue(std::int32_t[] { 3 }));
-            Assert::AreEqual(3, setter.Result);
-            Assert::AreEqual(0, setter.SetFirstAndReturnFalse(std::int32_t[] { 4 }));
-            Assert::AreEqual(4, setter.Result);
-        }
-    };
-}
+﻿#include <gtest/gtest.h>
+#include <Platform.Setters.h>
+namespace Platform::Setters::Tests
+{
+    TEST(SettersTests, ParameterlessConstructedSetterTest)
+    {
+        auto setter = Setter<std::int32_t>();
+        ASSERT_EQ(0, setter.Result());
+    }
+
+    TEST(SettersTests, ConstructedWithDefaultValueSetterTest)
+    {
+        auto setter = Setter<std::int32_t>(9);
+        ASSERT_EQ(9, setter.Result());
+    }
+
+    TEST(SettersTests, MethodsWithBooleanReturnTypeTest)
+    {
+        auto setter = Setter<std::int32_t>();
+        ASSERT_TRUE(setter.SetAndReturnTrue(1));
+        ASSERT_EQ(1, setter.Result());
+        ASSERT_FALSE(setter.SetAndReturnFalse(2));
+        ASSERT_EQ(2, setter.Result());
+        ASSERT_TRUE(setter.SetFirstAndReturnTrue(std::to_array<std::int32_t>({3})));

```

Comment:
Brackets combo!
You can at first create array then forward array to function
```suggestion
        auto array = std::to_array<std::int32_t>({3});
        ASSERT_TRUE(setter.SetFirstAndReturnTrue(array));
```